<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BFMessageBox</class>
 <widget class="QDialog" name="BFMessageBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BFMessageBox</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="main_wid" native="true">
     <property name="styleSheet">
      <string notr="true">#main_wid
{
background-color:#ff1c1d31;
border-radius:10px;
border:1px solid;
border-color:#ff303642;
}

</string>
     </property>
     <widget class="QPushButton" name="cancel_btn">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>180</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(225, 225, 225);</string>
      </property>
      <property name="text">
       <string>否</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirm_btn">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(225, 225, 225);
</string>
      </property>
      <property name="text">
       <string>是</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ok_btn">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>180</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(225, 225, 225);</string>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
     <widget class="QLabel" name="tips_lab">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>187</width>
        <height>26</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>提示</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_context">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>321</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_close">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>20</y>
        <width>15</width>
        <height>15</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
